### PR TITLE
Feat: create leaderboard design

### PR DESCRIPTION
**Pull request by Maksym45**
make style.css for site, item.js for display data about each player, List.js takes an array of data as a prop and renders a list of <Item> components, edit index.js, make data.json is data from database, index.js imports the data.json file, move package.json to folder src, because site did not want to launch.

### CODE REVIEW

File: react-app/src/data/data.json
Comment (StpdSerega) [inline react-app/src/data/data.json:1]: What is this file for? User data will be taken from the back

File: react-app/src/data/index.js
Comment (StpdSerega) [inline react-app/src/data/index.js:1]: Don't call files like "index/data and etc", this is for all files. Name them to make it clear what they correspond to

**Review by StpdSerega — COMMENTED** [review]
Fix issues and than i will merge it

**Comment by StpdSerega** [comment]
> make style.css for site, item.js for display data about each player, List.js takes an array of data as a prop and renders a list of components, edit index.js, make data.json is data from database, index.js imports the data.json file, move package.json to folder src, because site did not want to launch.

You don't need to write long description, if you want to show something - leave comment in code

**Comment by Maksym45** [comment]
Fix issues, delete data folder because get data from back

**Pull request merged**